docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md

```diff
@@ -134,3 +134,96 @@ console data:import glossary
+```
+#### Configuration
+
+##### AclConfig
+- Add `'merchant-app-merchant-portal-gui'` to your `\Pyz\Zed\Acl\AclConfig::addMerchantPortalInstallerRules()`.
+
+##### MessageBrokerConfig
+- `'merchant-commands'`, `'merchant-app-events'`, and `'app-events'` to your `\Pyz\Zed\MessageBroker\MessageBrokerConfig::getDefaultWorkerChannels()`.
+
+### State-Machine configuration
+
+Checkout the ForeignPaymentProviderStateMachine01.xml file in the `vendor/spryker/sales-payment/config/Zed/Oms/StateMachine/` directory. This file contains the state-machine configuration for the payment provider. You can copy this file to your project and adjust it according to your needs.
+
+#### Configuring Transfers
+
+In Spryker we used the terms "Payout" and "Reverse Payout" for transferring money from the Marketplace to the Merchant and reverse the transfer respectively.
+
+In the context of Stripe in a Marketplace, you need to configure the transfers. The transfers are handled by the `MerchantPayoutCommandByOrderPlugin` and `MerchantPayoutReverseCommandByOrderPlugin` commands. These commands are responsible for transferring money from the Marketplace to the Merchant and reverse the transfer respectively when needed.
+
+You also need to define when this should happen. You have several options here which are default options provided by the OMS. The simplest solution is to set a state-machine-timeout for the `MerchantPayoutCommandByOrderPlugin` command. This will trigger the command after the timeout is reached. You can also define your own conditions and triggers for the command.
+
+You can also set up a cronjob that triggers the event for the transition when you have more sophisticated requirements e.g. transfer money to merchants every last friday of the month.
+
+
+### App Configuration
+
+The Stripe App is capable of handling payments in a Marketplace context. This can be configured in the AppStore Catalog by selecting the business model "Marketplace". When selecting the Marketplace business model, you need to pass the required configuration values for:
+- Stripe Account ID 
+- Stripe Publishable Key
+- Stripe Secret Key
+
+When you save the configuration an asynchronous message will be sent to your application to get details for Merchants on how to onboard to the Stripe App. The onboarding is required to let Stripe know which Merchants are part of the Marketplace and to handle the payments, transfer money, and reverse transfers accordingly.
+
+From this moment your customers can use Stripe to pay for their orders.
+
+### Merchant Onboarding Process
+
+To be able to transfer money from the Marketplace Account to the Merchants, Merchants need to be onboarded to the Stripe App. When you configured the Stripe App in Marketplace mode, the Merchant onboarding process is initiated. The process is as follows:
+
+- Merchants need login to the Merchant Portal.
+- Merchants need to open the "Payment Settings" section.
```

Comment:
`config/Zed/navigation.xml` has to be updated to show this page using example from `spryker/merchant-app-merchant-portal-gui`